packages/create-ripple-app/template/nuxt/nuxt.config.js

```diff
@@ -42,12 +42,7 @@ export default {
   */
   modules: [
     // https://www.npmjs.com/package/@dpc-sdp/ripple-nuxt-tide
-    '@dpc-sdp/ripple-nuxt-tide',
-    // we want to always disallow bots on the example site
-    ['@nuxtjs/robots', {
-      UserAgent: '*',
-      Disallow: '/'
-    }]
```

Comment:
`robots` for each site still needed I think.

Reply:
should it be optional though? Should it even be something that nuxt tide implements?